Makefile

```diff
@@ -194,22 +204,26 @@ ux-tarball: build-ux dist ## Build website asset tarball
 		--directory=src/js/engagement_view/build \
 		.
 
-.PHONY: lambdas
-lambdas: lambdas-rust lambdas-js lambdas-python ## Generate all lambda zip files
-
-.PHONY: lambdas-rust
-lambdas-rust: ## Build Rust lambda zips
-	$(DOCKER_BUILDX_BAKE) -f docker-compose.lambda-zips.rust.yml
-	docker-compose -f docker-compose.lambda-zips.rust.yml up
-
-.PHONY: lambdas-js
-lambdas-js: ## Build JS lambda zips
-	$(DOCKER_BUILDX_BAKE) -f docker-compose.lambda-zips.js.yml
-	docker-compose -f docker-compose.lambda-zips.js.yml up
-
-.PHONY: lambdas-python
-lambdas-python: ## Build Python lambda zips
-	./pants filter --target-type=python_awslambda :: | xargs ./pants package
+.PHONY: extract-lambda-zips
+extract-lambda-zips: extract-lambda-zips-js extract-lambda-zips-rust ## Extract zips from previously built Docker images for AWS Lambda
+
+.PHONY: extract-lambda-zips-rust
+extract-lambda-zips-rust: build-docker-services
+	docker run --rm \
+		--user "${UID}:${GID}" \
+		--volume "${PWD}/dist:/tmp/zips" \
+		--workdir /grapl/zips \
+		grapl/metric-forwarder-zip:${TAG} \
+		sh -c 'cp /grapl/zips/metric-forwarder.zip /tmp/zips/metric-forwarder-lambda.zip'
+
+.PHONY: extract-lambda-zips-js
+extract-lambda-zips-js: build-docker-services
```

Comment:
An unfortunate side effect of this dependency structure is that if (for whatever reason) you only wanted to build this single Javascript lambda, you will necessarily end up building every other container, along with all our Python artifacts.

Similar logic holds for the Rust lambdas.

Reply:
Yeah, this was the idea for having these separate targets in the first place, but I know there have been complaints about complexity with this system and this seemed like a decent opportunity to simplify all the things. I wasn't aware of this being used, but sounds like it's potentially useful for, as you said in another comment, only building when the sources have changed.